tests/e2e/test_option_requests.py

```diff
@@ -22,7 +22,7 @@ def test_regular_options_request(http_methods: List["Method"]) -> None:
     def handler() -> None:
         return None
 
-    with create_test_client(handler, openapi_config=None) as client:
+    with create_test_client(handler, openapi_config=None, debug=True) as client:
```

Comment:
Why? We don't have this elsewhere either

Reply:
was simply debugging this one, can remove it. it has 0 impact.

Reply:
fixed